main_test.go

```diff
@@ -186,4 +188,59 @@ func testTemplateEvalCSVStdIn(t *testing.T) {
 	if string(out) != outputSampleText {
 		t.Fatalf("Expected:\n%s\nGot:\n%s", outputSampleText, string(out))
 	}
-}
\ No newline at end of file
+}
+
+func testTemplateDelimiters(t *testing.T) {
```

Comment:
Is your test running a binary? It shouldn't.

Reply:
Its not, its running a command the very same way the other tests above